content/articles/introduction-to-php-graphic-techniques/index.md

```diff
@@ -0,0 +1,227 @@
+ Introduction To PHP Graphic Techniques
+### Introduction
+PHP has the inherent ability to create and manipulate graphics independently. The scope and capabilities of PHP graphic handling is quite deep and powerful. It does this with the aid of the GD library which currently supports image formats in jpeg, png, gif and wbmp. 
+This tutorial aims at providing a solid foundation on the first steps to take to start doing exploits with PHP graphics. If you’re as excited as I am, then let’s proceed. 

```

Comment:
This sentence here is not quite clear and it's hard to follow. Let's have a simple sentence.

For example `This tutorial aims to provide a solid foundation on the first steps to start doing exploits with PHP graphics.` is much better that what you currently have!